### ISSUE TITLE
I am unable to get this form to submit data

### ISSUE THREAD

**Issue by dan9126** [gophish/gophish#1765]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?:  0.7.1

Brief description of the issue:  I have tried everything I can to get this landing page to submit data collected, but it doesn't. opens show, clicks show, but no collected data

What are you expecting to see happen? : user name collected. Password collected if set to gather credentials.

What are you seeing happen? Opened items show, Clickss show, but no submitted data shows.  I built the page myself to avoid any infringement, I am NOT an HTML guy, so its probably pretty ugly. Don't cringe, but I started in MS Word, moved it to notepad++, pasted it into GoPHish, then tinkered with it to add method POST to form and the button which is an older style of button I gather. It seems to work fine, but no data collected. I did check out other posts about this and Google a bunch, but nothing fixed it. Thanks for any help you can give me. 

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
<html><head>
<meta http-equiv="Content-Type" content="text/html; charset=windows-1252"/>
<meta name="Generator" content="Microsoft Word 15 (filtered)"/>
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	margin-bottom:.0001pt;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2E74B5;
	font-weight:normal;}
span.MsoPlaceholderText
	{color:gray;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#2E74B5;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}
-->
</style>

</head>

<body lang="EN-US">

<div class="WordSection1" align="center">

<h1><img width="2112" height="127" id="Picture 1" src="http://fchap1884/static/CMCHeader.png"/></h1>

<h1><span style="font-size:48.0pt;line-height:107%"> </span></h1>

<h1><span style="font-size:48.0pt;line-height:107%">
<b>Webinar Registration Portal</b></span></h1>
<br/>
<br/>
<p class="MsoNormal" align="center" style="text-align:center&#39;"> </p>

<form align="center" action="" autocomplete="off" method="POST" name="RegForm">
  <label for="uname">User Name:</label><br/>
  <input type="text" id="uname" value=""/><br/>
  <label for="Password">Password:</label><br/>
  <input type="text" id="Password" value=""/><br/><br/>
  <input type="button" value="Submit Registration" id="Submit Registration" onclick="location=&#39;https://auth4.mycmc.com/vpn/tmindex.html&#39;"/>
</form>  

<p class="MsoNormal" align="center" style="text-align:center"> </p>



<p class="MsoNormal" align="center" style="text-align:center"> </p>

<p class="MsoNormal" align="center" style="text-align:center"> </p>

<p class="MsoNormal" align="center" style="text-align:center"> </p>

<p class="MsoNormal" align="center" style="text-align:center"> </p>

<p class="MsoNormal" align="center" style="text-align:center">
Please sign in above to receive your content. You may choose to view it on line or
receive a link by email.</p>

</div>




</body></html>
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 -  Created email template works fine
 * Step 2 - Created and tested landing page. Page displays, I can type in fields, redirect works after submit button pressed. Just no data.
 * Step 3 -Added post to form and button. Made sure fields had names (saw that in other issues) retested.

**Comment by jordan-wright** [comment]
Try adding a `name` field to each input element. Some of the issues [here](https://github.com/gophish/gophish/search?q=name+attribute&type=Issues) should point you in the right direction.

**Comment by dan9126** [comment]
I thought I had but it doesn't look like it, looking at my code. I'll do that first thing tomorrow and update this. Thanks for the quick response.

**Comment by dan9126** [comment]
So, I think i have the names in. The updated version of the form is shown below. This may just be because I know as much about HTML coding as my dog does about particle physics. I appreciate your help on this.  I do have a name in for each one(each input type, not label), which I just re-used the id for, I hope that's OK. I checked that the form method is POST, so hopefully that matches the requirements (another words, I don't need another change in the HTML someplace else to make POST work here). I  do get that the user clicked on the link, just no user data. I do have Capture Sumitted Data checked as well as Capture Passwords.  

<form align="center" action="" autocomplete="off" method="POST" name="RegForm">
  <label for="uname">User Name:</label><br/>
  <input type="text" id="uname" name="uname" value=""/><br/>
  <label for="Password">Password:</label><br/>
  <input type="text" id="Password" name="password" value=""/><br/><br/>
  <input type="button" value="Submit Registration" id="Submit Registration" onclick="location=&#39;https://auth4.mycmc.com/vpn/tmindex.html&#39;"/>
</form>

**Comment by jordan-wright** [comment]
The last thing you'll want to do is remove the `onclick` attribute in your submit button. Right now, that's overriding the form submission, just sending the user to the URL listed in that attribute.

**Comment by TheGroundZero** [comment]
What's the action you're defining in the button for?  
Also, there's encoded content in the location (`&#39;`).

````html
<input type="button" value="Submit Registration" id="Submit Registration" onclick="location=&#39;https://auth4.mycmc.com/vpn/tmindex.html&#39;"/>
```

Try with an actual `submit` button and remove the onclick-function.

````html
<input type="submit" value="Submit Registration" id="Submit Registration" />
```

More info:

- https://www.w3schools.com/tags/att_input_type_submit.asp
- https://www.w3schools.com/tags/att_button_type.asp
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit

**Comment by dan9126** [comment]
Thank you very much for your help on this.  Your suggestion did the trick.  I still am amazed at (and very grateful for) the speed of your response.  I got it all working with a submit button separate from the form element.  
As an aside, I have experienced lots of problems trying to import an Outlook created email, probably again, just my ignorance.  I still haven't gotten it working right, so I have been building emails from scratch.  As part of this process, I found I that the workaround that was suggested in issue #251 does work find.  As a possible caveat to others, I note that, oddly, the fix below did not work for me with .png files, only with .jpg. Again, I may have had an unseen typo, but if you are having problems with a embedding a png file, try saving it as jpg (Paint does this just fine), attaching the .jpg file and referencing it as shown below. It seems to work great with Outlook 2016.

````<p><img src="cid:mylogo.jpg"></p>````

Also, when I did the preview, I didn't see the image but did see the broken link emblem. To see the results, I had to make a campaign and send myself the email. As a VERY new HTML person, I had a lot of test campaigns. 

In any case you did a great job supporting this which is much appreciated. This is a great help to the cybersecurity community and much appreciated. Many thanks all around.
